app/src/main/java/org/fossasia/openevent/general/aboutEvent/AboutEventActivity.kt

```diff
@@ -0,0 +1,99 @@
+package org.fossasia.openevent.general.aboutEvent
+
+import android.arch.lifecycle.Observer
+import android.os.Bundle
+import android.support.design.widget.AppBarLayout
+import android.support.v7.app.AppCompatActivity
+import android.view.MenuItem
+import android.view.View
+import android.widget.Toast
+import kotlinx.android.synthetic.main.fragment_about_event.*
+import org.fossasia.openevent.general.R
+import org.fossasia.openevent.general.event.Event
+import org.koin.android.architecture.ext.viewModel
+import java.lang.StringBuilder
+
+class AboutEventActivity : AppCompatActivity(), AppBarLayout.OnOffsetChangedListener {
+
+    private val aboutEventViewModel by viewModel<AboutEventViewModel>()
+    private var id: Long = -1
+    private val EVENT_ID: String = "EVENT_ID"
+    private var isHideToolbarView: Boolean = false
+    private lateinit var eventExtra: Event
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.fragment_about_event)
+
+        setSupportActionBar(aboutEventToolbar)
+
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
+        aboutEventCollapsingLayout.title = " "
+        appbar.addOnOffsetChangedListener(this)
+
+        aboutEventViewModel.error.observe(this, Observer {
+            Toast.makeText(this, it, Toast.LENGTH_LONG).show()
+        })
+
+        id = intent.getLongExtra(EVENT_ID, -1)
+
+        aboutEventViewModel.progressAboutEvent.observe(this, Observer {
+            it?.let { showProgressBar(it) }
+        })
+
+        aboutEventViewModel.loadEvent(id)
+
+        aboutEventViewModel.event.observe(this, Observer {
+            it?.let {
+                loadEvent(it)
+            }
+        })
+    }
+
+    private fun loadEvent(event: Event){
+        eventExtra = event
+        aboutEventContent.text = event.description
+        val dateString = StringBuilder("")
```

Comment:
No need to initialize StringBuilder with empty string

Reply:
Did the changes

Reply:
@nikit19 @simarsingh24 @iamareebjamal Please check whether any other modifications are required :+1:

Reply:
@dreadpool2 Can you rename the package to about. Since everything is about the events so aboutEvent does not convey much meaning according to me. It would also look similar to other packages like search,ticket etc

Reply:
Seems legit :+1:  Any other changes required ?